.devcontainer/cuda11.8-pip/devcontainer.json

```diff
@@ -0,0 +1,49 @@
+{
+  "build": {
+    "context": "${localWorkspaceFolder}/.devcontainer",
+    "dockerfile": "${localWorkspaceFolder}/.devcontainer/Dockerfile",
+    "args": {
+      "CUDA": "11.8",
+      "PYTHON_PACKAGE_MANAGER": "pip",
+      "BASE": "rapidsai/devcontainers:25.02-cpp-cuda11.8-ucx1.15.0-openmpi-ubuntu22.04"
```

Comment:
This uses ucx 1.15 but the CUDA 12 container uses ucx 1.17. Is that right?

Reply:
That's a great catch! I think it should be 1.17 here too.

I copied these directly from `cugraph`, looks like that was an oversight in https://github.com/rapidsai/cugraph/pull/4562#issuecomment-2258608174.

Let's try 1.17 here.

Reply:
We need to fix this in cugraph and nx-cugraph. https://github.com/search?q=org%3Arapidsai%20ucx1.15&type=code

Reply:
I just pushed a commit adding those changes (setting `RAPIDS_LIBUCX_PREFER_SYSTEM_LIBRARY=true` in the pip devcontainers) here.

And manually cancelled CI because it'll fail until https://github.com/rapidsai/raft/pull/2513 is done.